docs/notes/feature-layercliprect.md

```diff
@@ -0,0 +1,13 @@
+# New layerClipRect control property
+
+A new property 'layerClipRect' has been added to all controls.
+
+The property defines a clipping rectangle in card co-ordinates which
```

Comment:
I suggest just copying the revised description above and ditching the second paragraph here - as container layerMode lifts the restriction in a more general way.